### ISSUE TITLE
wise need more docs: docs, docs, docs

### ISSUE THREAD

**Issue by Gandalf-the-Grey** [noisy-witness/steem-wise-core#11]
Wise needs more docs for various groups of users.
ELI5 for voters, quickstart guide for easy adoption, in-depth guides for devs.

Put a user in front of a https://github.com/noisy-witness/steem-wise-core/
Alice wants to ~be~ use wise, she want to give Bob, Carlos and Dave a way to vote on her behalf.
Eve is a poweruser, she wants to contribute by creating rules.
Alice needs an ELI5+quickstart guide, easy steps on how to deploy it and use it.
Bob and Carlos an ELI5 guide on how to vote, Dave and Eve need a guide for devs.

Currently there's *some* info in README, with mixed level of complexity and target groups.

**Comment by Jblew** [comment]
Your suggestion was very important for us. Since it, we have already:
- written a detailed manual: https://wise.vote/introduction
- updated README in every repository. They are now very detailed.
- added CONTRIBUTING.md guide to every repository. They are now very detailed.
- created good and precise code documentation in steem-wise-core repository
- created "wise init" command in steem-wise-cli which guides delegators through the daemon setup process.
